support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/accesstoken/response/OAuth20JwtAccessTokenEncoder.java

```diff
@@ -49,13 +49,26 @@ public class OAuth20JwtAccessTokenEncoder implements CipherExecutor<String, Stri
 
     private final String issuer;
 
+    /**
+     * Doing basic check that {@code JWTParser.parse} does to reduce logging of stack traces.
+     * @param tokenId Possibly encoded token.
+     * @return true
+     */
+    private boolean isPossiblyEncoded(final String tokenId) {
+        return tokenId.contains(".");
```

Comment:
This is not strictly a problem, but somewhat dangerous. A proper JWT checking I think would have to split this by a "." and then check to see if the split count is 3.

Reply:
Apparently a an encrypted JWT token can have 4 dots and 5 parts or 2 dots and 3 parts.. just realized my fix is bad... will update.